src/store/macros/getters.ts

```diff
@@ -2,6 +2,14 @@ import type { GetterTree } from 'vuex'
 import type { Macro, MacrosState } from './types'
 import type { RootState } from '../types'
 
+export const MACRO_DEFAULTS = {
+  alias: '',
+  visible: true,
+  disabledWhilePrinting: false,
+  color: '',
+  categoryId: '0'
```

Comment:
Added missing order field

```suggestion
  categoryId: '0',
  order: undefined
```